### DISCUSSION TITLE
Setting up jekyll

### DISCUSSION THREAD

**Discussion by ksdavidc** [codesandbox/codesandbox-client#5463] [Q&A]
I am a bit of a newbie trying to set up a generic jekyll site...just enough to give me a preview of the site so i can test code, actually.

In my setup on my local machine all I need to do is run bundle install, bundle update, and bundle exec jekyll serve, and I head over to port:4000 and I can see the site. For the life of me I can't figure out how to do the equivalent in codesandbox. It seems like it must be possible, but I have tried different configuration files and the kitchen sink but without an working example I am stuck....

Thanks in advance for any help.

**Comment by garethx** [comment]
As far as I know, Jekyll needs Ruby support to run, but we only support Node.js in containers at this time.

**Comment by ksdavidc** [comment]
I see.

I am confused because when  I explored the templates and searched for jekyll there are many of them, but I wasn't able to get any of them to work. Am I missing something? 

![image](https://user-images.githubusercontent.com/62230004/107571377-2b8eb400-6c2e-11eb-8297-5da0cce677eb.png)

That being the case, can you recommend an alternative to codesandbox that might be able to do it?

**Reply by garethx** [reply]
No, I don't think you're missing anything - they all look like folks trying to see if it works. Note that projects imported from GitHub work as templates, and if you look at the projects, they're either assuming Jekyll is already installed or are trying to install it, which doesn't work.

**Comment by ksdavidc** [comment]
Thank you! I guess everyone made the same mistake I did. For the record, goorm.io works, but requires a bit of  poorly documented magic. I was able to find the answer in korean with a bit of google trickery.  Gitpod works too, but not on private reps in free version. Only ones I was able to make work. Some other popular ones (Stackbit, et. al.) I wasn't able to get to work at all, perhaps for the same reason, or just because I couldn't find a kind person like you to set me straight 😉  😄
